src/gui/sharemanager.cpp

```diff
@@ -146,10 +146,10 @@ void Share::deleteShare()
     job->deleteShare(getId());
 }
 
-bool Share::isShareTypeUserGroupEmailRoomOrRemote(const ShareType type)
+bool Share::isShareTypeUserGroupEmailRoomRemoteOrCircle(const ShareType type)
 {
     return (type == Share::TypeUser || type == Share::TypeGroup || type == Share::TypeEmail || type == Share::TypeRoom
-        || type == Share::TypeRemote);
+        || type == Share::TypeRemote || type == Share::TypeCircle);
```

Comment:
IMO these should each be on a new line for readability

Reply:
done